src/oc_erchef/apps/chef_index/src/chef_elasticsearch.erl

```diff
@@ -25,6 +25,12 @@ ping() ->
 update(Body) when is_list(Body) ->
     update(iolist_to_binary(Body));
 update(Body) ->
+    io:format("Body:~p~n", [Body]),
+    %% Try to get labels to pass
+    prometheus_counter:declare([{name, count_es_post_update},
```

Comment:
Can we move these `delcare` statements somewhere where they will only happen once?  Perhaps when we start the chef_index_sup?